source/MaterialXGenShader/Util.cpp

```diff
@@ -455,91 +455,80 @@ bool elementRequiresShading(const TypedElementPtr element)
             colorClosures.count(elementType) > 0);
 }
 
-void findRenderableElements(const DocumentPtr& doc, std::vector<TypedElementPtr>& elements, bool includeReferencedGraphs, 
-                            std::ostream* errorLog)
+void findRenderableElements(const DocumentPtr& doc, std::vector<TypedElementPtr>& elements, bool includeReferencedGraphs)
 {
-    try
```

Comment:
Remove this try/catch as the errorLog passed in was always null and never used, causing the exceptions to be silently hidden below.